osmosis-1/osmosis.zone_chains.json

```diff
@@ -863,6 +863,18 @@
         "ibc-transfer",
         "ibc-go"
       ]
+    },
+    {
+      "chain_name": "aioz",
```

Comment:
Please define staking and fee rates (low, avg, high) in the [Chain Registry](https://github.com/cosmos/chain-registry/blob/master/aioz/chain.json)